FUNKI/data/reports/kinact_report.Rmd

```diff
@@ -0,0 +1,68 @@
+---
+title: "KinAct Analysis Report"
+output: html_document
+params:
+  minsize: NA
+  method: NA
+  organism: NA
+  selected_kinase: NA
+  selected_sample: NA
+  numberKinases: NA
+  sample_plot: NA
+  tf_plot: NA
+  network_plot: NA
+  heatmap_plot: NA
+---
+
+[KinAct](http://saezlab.github.io/kinact/) is a resource linking kinases to phosphorylation sites ( [Wirbel et al., 2018](https://www.biorxiv.org/content/10.1101/066019v1) ). 
```

Comment:
Are you sure that the biorxiv version is the most recent publication about kinact?

Reply:
fixed with the last commits